imagersite/imager_profile/views.py

```diff
@@ -0,0 +1,42 @@
+"""Views for user profiles."""
+
+
+from django.shortcuts import render
+from django.contrib.auth.models import User
+from django.conf import settings
+from imager_profile.models import ImagerProfile
+from imager_images.models import Photo, Album
+
+
+def profile_view(request):
+    """View for the logged-in user's profile."""
+    user = request.user
+    albums = Album.objects.all().filter(created_by=user)
```

Comment:
This doesn't work and throws and error, this is because you are trying to filter by a User model, not the Profile model which is what the `created_by` attribute is expecting.